Finmer.Game/Views/CharCreateBasic.xaml.cs

```diff
@@ -35,15 +35,30 @@ private void CharCreateViewBase_Loaded(object sender, RoutedEventArgs e)
         {
             txtName.Text = InitialSaveData.GetString("name");
             cmbSpecies.Text = InitialSaveData.GetString("species");
-            optGenderMale.IsChecked = InitialSaveData.GetString("gender").Equals("Male");
-            optGenderFemale.IsChecked = !optGenderMale.IsChecked;
+            if (!String.IsNullOrWhiteSpace(InitialSaveData.GetString("gender")))
+            {
+                optGenderMale.IsChecked = InitialSaveData.GetString("gender").Equals("Male");
+                optGenderFemale.IsChecked = !optGenderMale.IsChecked;
+            }
             m_Setup = false;
 
             // Apply some default settings to enable quickly clicking through to the game
             //if (GameController.DebugMode)
             {
                 txtName.Text = "Snack";
                 cmbSpecies.SelectedIndex = CoreUtility.Rng.Next(cmbSpecies.Items.Count);
+                //make a coin flip to see if it's male or female randomly selected, then set the gender accordingly.
+                int genderFlip = CoreUtility.Rng.Next(2);
+                if (genderFlip == 0) 
+                {
+                    optGenderMale.IsChecked = true;
+                    optGenderFemale.IsChecked = !optGenderMale.IsChecked;
+                }
+                else
+                {
+                    optGenderMale.IsChecked = false;
+                    optGenderFemale.IsChecked = !optGenderMale.IsChecked;
+                }
```

Comment:
I think this can be expressed more succinctly by checking the coin flip in the assignments directly:
```suggestion
                optGenderMale.IsChecked = (coin_flip == 0);
                optGenderFemale.IsChecked = !optGenderMale.IsChecked;
```